config/locales/en.yml

```diff
@@ -4,10 +4,10 @@ en:
     account_reset_cancellation_notice: Your request to delete your login.gov account has been cancelled.
     account_reset_notice: As requested, your login.gov account will be deleted in 24 hours. Don't want to delete your account? Sign in to your login.gov account to cancel.
     authentication_otp:
-      sms: Enter %{code} in login.gov to continue signing in to your account. This security code will expire in %{expiration} minutes.
+      sms: 'Login.gov: Your security code is %{code}. It expires in %{expiration} minutes. Don''t share this code with anyone.'
       voice: Hello! Your login.gov one time passcode is, %{code}, again, your passcode is, %{code}. This code expires in %{expiration} minutes.
     confirmation_otp:
```

Comment:
Now that these keys are the same, can we remove one and just have both methods reference the same translations?

Reply:
I think if we make the voice messages the same, but doesn't seem worth redoing the existing logic: https://github.com/18F/identity-telephony/blob/7661a8cf887bbb9383dde27bdcab306daccda37d/lib/telephony/otp_sender.rb#L57-L71